src/test/opentrackIO_parser/python/opentrackIO_parser.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+#
+# opentrackIO_parser.py
+#
+# Reference code for parsing/decoding opentrackIO samples
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the SMTPE RIS OSVP Metadata Project
+#
+# Example run: python3 opentrackIO_parser.py --file=opentrackio_complete_static_example-20241001.json --schema=opentrackio_schema_20241001.json
+# This is tested against the generated "complete_static_example" and "complete_dynamic_example" json
+
+import os
+import argparse
+from opentrackIOlib import *
+
+parser = argparse.ArgumentParser(description='OpenTrackingProtocol parser')
```

Comment:
Best practice is to encapsulate in a main() function (otherwise everything is effectively global)
```
def main():
    print("Hello World!")

if __name__ == "__main__":
    main()
```

Reply:
Since this is a fork and not a branch, I will merge this PR and then do this on my branch